### PR TITLE
doc(1.8.0): add DR volumes support for v2 data engine

### PR DESCRIPTION
**Pull request by c3y1huang**
#### Which issue(s) this PR fixes:

Issue longhorn/longhorn#6613

#### What this PR does / why we need it:

Add DR volume support for the v2 data engine in the important note.

#### Special notes for your reviewer:

`None`

#### Additional documentation or context

`None`

### CODE REVIEW

**Review by derekbit — DISMISSED** [review]
LGTM

File: content/docs/1.8.0/important-notes/_index.md [hunk 1]
```diff
@@ -226,6 +227,6 @@ Snapshots created before Longhorn v1.7.0 may change occasionally. This issue ari
 
 Reverting a volume to a snapshot created before Longhorn v1.7.0 is not supported due to an incorrect UserCreated flag set on the snapshot. The workaround is to back up the existing snapshots before upgrading to Longhorn v1.7.0 and restore them if needed. The bug is fixed in v1.7.0, and more information can be found [here](https://github.com/longhorn/longhorn/issues/9054).
 
+### Disaster Recovery Volumes
 
-
-
+Disaster recovery volumes is supported since Longhorn v1.8.0.
```
Comment (jhkrug) [inline content/docs/1.8.0/important-notes/_index.md:232]:
```suggestion
Disaster recovery volumes are supported from Longhorn v1.8.0.
```

**Review by jhkrug — CHANGES_REQUESTED** [review]
A small suggestion.

File: content/docs/1.8.0/important-notes/_index.md (hunk 1 above)
Comment (c3y1huang) [reply content/docs/1.8.0/important-notes/_index.md]: Updated, thank you!

**Review by derekbit — APPROVED** [review]
LGTM

**Review by derekbit — APPROVED** [review]

**Pull request merged**